hash/vsmile_cart.xml

```diff
@@ -2821,8 +2821,8 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 	</software>
 
 	<software name="noddyfrr105" cloneof="noddy">
-		<description>Oui-Oui - Détective d'un Jour (France, rev. 105)</description>
```

Comment:
why are you changing Oui-Oui to Noddy, it's clearly Oui Oui on the French title screens.